auth0-extensions/auth0-logs-to-provider
# Bump Express to 4.17.1

## ✏️ Changes
  
Webtask has Express 4.17.1 installed, not 4.17.0. This tweaks the package version.

   
## 📷 Screenshots


```
{
  "code": 400,
  "message": "Compilation failed: Cannot find module 'express@4.17.0'",
  "error": "Cannot find module 'express@4.17.0'",
  "stack": "Error: Cannot find module 'express@4.17.0'\n    at Function.Module._resolveFilename (module.js:548:15)\n    at Function.WebtaskModule._resolveFilename (/data/sandbox/lib/module.js:96:27)\n    at WebtaskModule.require (/data/sandbox/lib/module.js:149:39)\n    at require (/data/sandbox/lib/module.js:184:21)\n    at Object.<anonymous> (/data/io/06d3ed38-503a-417c-a30a-2fd5d6c58af5/webtask.js:1:2271)\n    at t (/data/io/06d3ed38-503a-417c-a30a-2fd5d6c58af5/webtask.js:1:177)
```
  
## 🔗 References
  
See that express 4.17.0 is missing here: https://auth0-extensions.github.io/canirequire/#express
  
## 🎯 Testing
  
Run dev server locally:

1. Git clone auth0/auth0-extensions
2. `git checkout an approrpriate branch`
3. `npx http-server --port 3001 --cors`

Use Chrome to test locally:

1. Install Chrome Browser extension
2. Update extension to use your local dev server
2. Login to auth0.com
3. Enable extension
4. Visit Extensions page in Dashboard
5. Confirm the extension has the correct version
6. Install
7. Click on it
8. If it is broke it will crash with 4.17.0 -- the fixed version will work
   
🚫 This change has been tested in a Webtask
  
## 🚀 Deployment
  
> Can this change be merged at any time? What will the deployment of the change look like? Does this need to be released in lockstep with something else?
  
✅ This can be deployed any time
 
  
## 🎡 Rollout
  

  
## 🔥 Rollback
  

### 📄 Procedure
  
 
## 🖥 Appliance
